packages/destination-actions/src/destinations/tiktok-audiences/properties.ts

```diff
@@ -0,0 +1,91 @@
+import { InputField } from '@segment/actions-core/src/destination-kit/types'
+
+export const selected_advertiser_id: InputField = {
+  label: 'Advertiser ID',
+  description: 'The advertiser ID to use when syncing audiences.',
+  type: 'string',
+  dynamic: true,
+  required: true
+}
+
+export const custom_audience_name: InputField = {
+  label: 'Custom Audience Name',
+  description:
+    'Custom audience name of audience to be created. Please note that names over 70 characters will be truncated to 67 characters with "..." appended. This field is set only when Segment creates a new audience. Updating this field after Segment has created an audience will not update the audience name in TikTok.',
```

Comment:
Should we say something like:

```suggestion
    'Custom audience name of audience to be created. Please note that names over 70 characters will be truncated to 67 characters with "..." appended. This field is set only when Segment creates a new audience. Updating this field after Segment has created an audience will not update the audience name in TikTok. Instead, updating the audience name here will create a separate audience in TikTok with the new audience name. ',
```